### ISSUE TITLE
Add a `question.md` Issue Template for Enhanced Community Engagement

### ISSUE THREAD

**Issue by Umpire2018** [OpenDevin/OpenDevin#1114]
**Summary**
I would like to propose the addition of a new issue template for questions. This can help organize and streamline the process of asking questions in this repository.

**Motivation**
Having a structured format for questions will make it easier for contributors to provide the necessary details when seeking information or clarification. It ensures that questions are clear and concise, which can lead to more efficient communication and quicker responses.

Thank you for considering this proposal.

**Comment by Umpire2018** [comment]
@rbren  I have a question I'd like to raise. I'm currently using `make start-backend` but found a authentication issue.

### Details 
The code looks like this:

```python
@app.websocket('/ws')
async def websocket_endpoint(websocket: WebSocket):
    await websocket.accept()
    sid = get_sid_from_token(websocket.query_params.get('token') or '')
    if sid == '':
        return
    session_manager.add_session(sid, websocket)

def get_sid_from_token(token: str) -> str:
    """Gets the session id from a JWT token."""
    try:
        payload = jwt.decode(token, JWT_SECRET, algorithms=['HS256'])
        if payload is None:
            logger.error('Invalid token')
            return ''
        return payload['sid']
    except Exception as e:
        logger.exception('Error decoding token: %s', e)
        return ''
```

### Question

Is there a way to manually obtain or construct the JWT needed for this? I'm looking for a method to generate or retrieve a valid JWT to test the websocket connection manually.

**Comment by rbren** [comment]
@Umpire2018 I'm not sure on the answer here--you might want to try asking the person who created the JWT logic (you can see it using the blame UI). But let's move that discussion to a new issue.

Totally agree on this issue though--a question template would be great

**Comment by Umpire2018** [comment]
> @Umpire2018 I'm not sure on the answer here--you might want to try asking the person who created the JWT logic (you can see it using the blame UI). But let's move that discussion to a new issue.
> 
> Totally agree on this issue though--a question template would be great

Thank you. Once #1115 is merged, i will open a new issue.
